assets/dev/scss/frontend/_compatibility.scss

```diff
@@ -19,6 +19,7 @@
 	padding: 0;
 }
 
+// To be removed in Elementor 3.26.0 or 3.27.0
```

Comment:
We will need this styling in this filing, because of a Core/Pro dependency.

I am not sure if there another way to handle this.